### ISSUE TITLE
`runtimeConfig` в `nuxt` не принимается во внимание

### ISSUE THREAD

**Issue by zumm** [yandex-maps-unofficial/vue-yandex-maps#345]
### Версия библиотеки

2.1.4

### Как воспроизвести

Использовать `vue-yandex-maps/nuxt` модуль и задать api ключ через env или через `runtimeConfig` в `nuxt.config.ts`, а не напрямую через конфигурацию модуля.

### Опишите проблему

Модуль для `nuxt` игнорирует `runtimeConfig`. Подозреваю из-за вот этой перезаписи: https://github.com/yandex-maps-unofficial/vue-yandex-maps/blob/master/packages/vue-yandex-maps/src/plugins/nuxt3-module.ts#L59

### Ответы на вопросы

- [X] Я использую версию 2.x
- [X] Я поискал похожие ошибки и не нашел таковых

**Comment by PNKBizz** [comment]
Вот так работает
```env
NUXT_PUBLIC_YANDEX_MAPS_APIKEY=xxxx
```
```ts
export default defineNuxtConfig({
  modules: ['vue-yandex-maps/nuxt'],
  yandexMaps: {
    apikey: ''
  },
  runtimeConfig: {
    public: {
      yandexMaps: {
        apikey: ''
      }
    }
  }
})
```

**Comment by zumm** [comment]
Да, но если не указывать пустой ключ в конфигурации, то не работает. Это запланированное поведение накст модулей?

```js
export default defineNuxtConfig({
  modules: ['vue-yandex-maps/nuxt'],
  runtimeConfig: {
    public: {
      yandexMaps: {
        apikey: ''
      }
    }
  }
})
```

**Comment by daniluk4000** [comment]
> Да, но если не указывать пустой ключ в конфигурации, то не работает. Это запланированное поведение накст модулей?
> 
> ```js
> export default defineNuxtConfig({
>   modules: ['vue-yandex-maps/nuxt'],
>   runtimeConfig: {
>     public: {
>       yandexMaps: {
>         apikey: ''
>       }
>     }
>   }
> })
> ```

Да, так оно работает в Nuxt 3

**Comment by daniluk4000** [comment]
Давайте я что нибудь попробую с этим сделать в следующей мажорке
